main_test.go

```diff
@@ -49,3 +49,20 @@ func TestFindRoot(t *testing.T) {
 		}
 	}
 }
+
+func TestSlashedGOPATH(t *testing.T) {
+	gopath := os.Getenv("GOPATH")
```

Comment:
does this introduce any kind of potential problems if we ever try to run tests in parallel? (i genuinely don't know, and am asking)